dbgpt/rag/transformer/summary_triplet_extractor.py

```diff
@@ -0,0 +1,20 @@
+"""TripletExtractor class."""
+import logging
+
+from dbgpt.core import LLMClient
+from dbgpt.rag.transformer.triplet_extractor import TripletExtractor
+
+logger = logging.getLogger(__name__)
+
+SUMMARY_TRIPLET_EXTRACT_PT = (
+    # TODO: provide prompt template here
+)
+
+
+class SummaryTripletExtractor(TripletExtractor):
```

Comment:
@M1n9X 
The implementation code of triple extraction with element summary is placed here.